src/models/setup-context.ts

```diff
@@ -0,0 +1,30 @@
+import { callApiMethod } from '@api/entrypoint';
+import { scopedLogger } from '@logger';
+import { getPrependScript } from '@sandboxed/prepare-scripts';
+import { registerAsyncFunction, registerFunction } from '@utils/registerFunc';
+import { Context } from 'isolated-vm';
+import joi from 'joi';
+
+const log = scopedLogger('sandbox');
+const stringSchema = joi.string().required().strict();
+
+export async function createGlobalContext(ctx: Context) {
+  const jail = ctx.global;
+  await jail.set('global', jail.derefInto());
+
+  // simple console log for debugging purposes
+  // TODO remove this function from production code
+  await registerFunction(ctx, 'log', (msg: any) => {
+    if (stringSchema.validate(msg).error) return false;
+    log.debug('SANDBOX: ' + msg);
+    return true;
+  });
```

Comment:
No reason we can't have this log in prod. Just make prod not print debug logs. We're going to need to add more stuff here for sending logs to the dashboard or other system, but I think it's fine having debug logging sandbox messages.

Reply:
dont sure I agree, we'll talk with binary about it